data/datapackage.yaml

```diff
@@ -0,0 +1,338 @@
+        - name: book_id
+          type: string
+          format: uri
+          description: Link to the book where the reference was found
+          constraints:
+            required: true
+        - name: book_page
+          type: string
+          description: Page of the book in Derrida's citation
+        - name: book_type
+          type: string
+          description: Whether or not the reference is to a book or journal article
+          constraints:
+            required: true
+            pattern: Book|Journal Article
+        - name: anchor_text
+          type: string
+          description: The text of the quotation or the footnote that makes up the reference
+        - name: interventions
+          type: any
+          description: A URI to the linked annotation (if Derrida had a book where this reference was annotated.)
+          # ?: Is this accurate
+        - name: section
+          type: string
+          description: Section of De la grammatologie where the reference is located
+          constraints:
+            pattern: Part 1|Part 2
+          constraints:
+            required: true
+        - name: chapter
+          type: string
+          description: Chapter of De la grammatologie where the reference is located
+    scheme: file
+  - encoding: utf-8-sig
+    format: csv
+    hashing: md5
+    name: library
+    path: library.csv
+    description: Books from Derrida's personal library
+    profile: tabular-data-resource
+    schema:
+      primaryKey: id
+      foreignKeys:
+        # Frictionless Data doesn't seem to run tests on self-referencing foreign keys.
+        #  Whether all of collected_work_uri exists in the id column was checked manually.
+        #  When introducing an error into the dataset, Frictionless Data could not detect it.
+        #  https://specs.frictionlessdata.io/table-schema/#foreign-keys
+        - fields: collected_work_uri
+          reference:
+            resource: ""
+            fields: id
+      fields:
+        - name: id
+          type: string
+          format: uri
+          description: Unique link to the book's detail page
+          constraints:
+            required: true
+            unique: true
+        - name: item_type
+          type: string
+          description: The type of work (either a book, journal article, or book section)
+          constraints:
+            pattern: Book|Journal Article|Book Section
+          constraints:
+            required: true
+        - name: title
+          type: string
+          description: Title of the given work
+          constraints:
+            required: true
+        - name: alternate_title
+          type: any
+          description: '' # This means something specific, right?
+        - name: work_year
+          type: integer
+          description: Year in which the work was published
+        - name: copyright_year
+          type: integer
+          description: Year in which the work was copyrighted
```

Comment:
```suggestion
          description: Copyright year for this work
```

Reply:
"in which ... was copyrighted" sounds so stilted; may want to revise "was published" also. do we need to specify "this work?"

Reply:
No, we shouldn't have to specify, and then it becomes "Copyright year." Do I have permission to just delete this description?

Reply:
yes!

Reply:
✅ 4264db817c2c610c04f0b41e210de7f2a25ba798